### ISSUE TITLE
Experiments on VOC split 1

### ISSUE THREAD

**Issue by GuangxingHan** [ucbdrive/few-shot-object-detection#52]
In my experiment, the results on VOC split 1 is much lower than the paper reports. 
For example, the mAP50 on the novel set is 
shot              |    1      |     5
paper            |  39.8  |  55.7
my result      |  34.1   |  49.1

However, the results on split 2 & 3 is similar to the paper report. I use the default params and script for training. What do I need to do to reproduce your results?

**Comment by Ze-Yang** [comment]
I also get the same results with @GuangxingHan. For TFA w/ fc, I only get 34.86 while the paper reports 36.8. I indeed use the official codebase without any modification. Plus, I download the pretrained model in the model zoo that the author provides.

**Comment by Ze-Yang** [comment]
@GuangxingHan Have you try with the model the author released? I get the similar split1 5shot results with the reported one, ~ 55% mAP on novel classes. However, if I train the base model by myself, it will degrade to ~52%. Do you have such problem? Thanks.

**Comment by GuangxingHan** [comment]
Yes, this is same to me. The official released model have better accuracy about 55% mAP on novel class. But if I train it by myself, the result is only about 50%.

**Comment by geonuk-kimmm** [comment]
I finally got mAP almost same with the reported ones. I found that batchsize and learning rate have to be well selected. If you have less gpu than original setting, you would have to tune the hyperparams.

**Comment by Ze-Yang** [comment]
@mandal4 I indeed ran the code with 8 GPUs as the auther's default setting. Could you provide your hyperparams?

**Comment by zzzzjx** [comment]
> 
Can you get the same performerce with paper with less gpu than original setting? Could you please tell me how to tune your hyperparams?  Thanks!

**Comment by zzzzjx** [comment]
> I finally got mAP almost same with the reported ones. I found that batchsize and learning rate have to be well selected. If you have less gpu than original setting, you would have to tune the hyperparams.

Can you get the same performence with the paper? Please share me how to  tune the hyperparams with less gpu than original setting? Thanks very much!
